src/api/todos.ts

```diff
@@ -0,0 +1,15 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const addTodo = (userId: number, data: any) => {
```

Comment:
dont use 'any ' for type

Reply:
What type to use instead? It's not a Todo type because it lacks an id.

Reply:
if you don't want to get a primitive type, you can use object